js/ui/components/clubhouse.js

```diff
@@ -147,10 +159,22 @@ var ClubhouseComponent = new Lang.Class({
         this._clearBanner();
 
         if (this._clubhouseProxyHandler > 0)
-            this._clubhouseProxy.disconnect(this._clubhouseProxyHandler);
+            this.proxy.disconnect(this._clubhouseProxyHandler);
 
+        this.parent();
         this._clubhouseSource = null;
-        this._clubhouseProxy = null;
+    },
+
+    notifyHideAnimationCompleted: function() {
```

Comment:
Does not look like this method exists on the clubhouse interface.

Reply:
It was a left over from C&P... Thanks.